plugin.video.twitch/addon.xml

```diff
@@ -23,22 +25,22 @@
         <source>https://github.com/MrSprigster/Twitch-on-Kodi</source>
         <license>GNU GENERAL PUBLIC LICENSE Version 3, 29 June 2007</license>
         <forum>http://forum.kodi.tv/showthread.php?tid=134538</forum>
-        <summary lang='da_dk'>Twitch video plugin</summary>
-        <description lang='da_dk'>Se dine favorit gaming streams!</description>
-        <summary lang='de_de'>Twitch video plugin</summary>
-        <description lang='de_de'>Schaue die besten Gaming-Streams!</description>
-        <summary lang='en_gb'>Twitch video plugin</summary>
-        <description lang='en_gb'>Watch your favorite gaming streams!</description>
-        <summary lang='es_es'>Twitch video plugin</summary>
-        <description lang='es_es'>¡Mira los streams de tus juegos favoritos!</description>
-        <summary lang='fr_fr'>Twitch video plugin</summary>
-        <description lang='fr_fr'>Regardez les streams de vos jeux favoris !</description>
-        <summary lang='nl_nl'>Twitch video plugin</summary>
-        <description lang='nl_nl'>Bekijk je favoriete gaming-streams!</description>
-        <summary lang='nb_no'>Tillegg for Twitch</summary>
-        <description lang='nb_no'>Se dine favorittstrømmere!</description>
-        <summary lang='pl_pl'>Twitch video plugin</summary>
-        <description lang='pl_pl'>Oglądaj ulubione programy TwitchTV!</description>
+        <summary lang='en_GB'>Twitch video plugin</summary>
+        <description lang='en_GB'>Watch your favorite gaming streams!</description>
```

Comment:
Moving en_GB to first in list because the add-on showcase seems to use en_GB for summary but the first description.
Can be seen https://kodi.tv/addon/plugins-video-add-ons/twitch